Pipfile

```diff
@@ -0,0 +1,11 @@
+[[source]]
```

Comment:
It looks like these were made in the root directory. You should ideally run your virtualenv from the server directory to make sure you're adding things to the right pipfile and running things properly. So you can delete this `Pipfile` and `Pipfile.lock`

Reply:
Ticket 17 - PR changes is up and ready for review